src/Avalonia.Themes.Fluent/Controls/ComboBox.xaml

```diff
@@ -125,7 +136,6 @@
                       Background="{DynamicResource ComboBoxDropDownBackground}"
                       BorderBrush="{DynamicResource ComboBoxDropDownBorderBrush}"
                       BorderThickness="{DynamicResource ComboBoxDropdownBorderThickness}"
-                      Margin="0,-1,0,-1"
```

Comment:
I removed this line as it was causing the `ComboBox` popup border to be truncated at the top and bottom. I've noticed this for a long time but never investigated it. @danwalmsley you seem to have added this line, do you remember why and it is ok to remove it?

Reply:
That is a copy-paste from Fluent v1 in WinUI

https://github.com/microsoft/microsoft-ui-xaml/blob/7ed0bcf32a0e02b673a3d29555bd604d1f9ac0b0/dev/ComboBox/ComboBox_themeresources_v1.xaml#L721

That Margin (and any others like it) can safely be removed. It was just a work-around for WinUI/UWP specific issues.